web/src/views/web/page/inventory/join.ejs

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <%- include('../../layout/head.ejs') -%>
+</head>
+<body>
+<%- include('../../layout/header.ejs') -%>
+
+<main>
+    <div class="hero">
+        <h1><%= title %></h1>
+        <table>
+            <thead>
+            <tr>
+                <th>Serial Num</th>
+                <th>Container ID</th>
+                <th>Type Name</th>
+                <th>Weight</th>
+                <th>Manufacture Date</th>
+            </tr>
+            </thead>
+            <tbody>
+            <% if (inventoryList) {%>
+                <% inventoryList.forEach(i => { %>
+                    <tr>
+                        <td><%= i.serial_num %></td>
+                        <td><%= i.container_id %></td>
+                        <td><%= i.type_name %></td>
+                        <td><%= i.weight %></td>
+                        <td><%= i.getDateString() %></td>
+                    </tr>
+                <% }); %>
+            <% } %>
+            </tbody>
+        </table>
+
+        <p>This is the result of the following query:</p>
+        <pre>
+  SELECT I.serial_num, I.container_id, I.type_name, I.weight, I.manufacture_date
+  FROM inventory I, storage_container S WHERE S.id = I.container_id AND S.id = < Given Container ID >;</pre>
```

Comment:
maybe put on a new line :)

```suggestion
  FROM inventory I, storage_container S
  WHERE S.id = I.container_id AND
        S.id = <Given Container ID>;</pre>
```

You can make this change along with adding a button to /inventory at the site's index

Reply:
Might also be nice to have a `style="button button-outline"` button on the query page to return to `/inventory`.

Reply:
Done, but through a separte commit as these changes are outdated